learnyounode/01-baby-steps.js

```diff
@@ -0,0 +1,12 @@
+sumNum = function(array) {
+  return array.reduce(function(pV, cV) {
+    return parseInt(pV) + parseInt(cV);
+  })
+};
+
+
+// sumNum(readLine(/\s/));
+
+// console.log(sumNum(process.argv.splice(0,2)));
+
+console.log(sumNum(process.argv.slice(2,this.length)));
```

Comment:
`.slice(2)` will also work. 

I would also suggest doing the `parseInt` step here, through `map`.
i.e. `process.argv.slice(2).map(parseInt)`
You could even chain a reduce on to that directly.
i.e. `process.argv.slice(2).map(parseInt).reduce((a,b) => a + b)`